app/assets/stylesheets/welcome.css.scss

```diff
@@ -0,0 +1,26 @@
+#banner {
+	background-image: url(image_path('welcome.jpg'));
+	background-size: cover;
+	background-repeat: no-repeat;
+	background-position: top center;
+	width: 100%;
+	min-height: 500px;
+	display: table;
+	.banner_content {
```

Comment:
Rule declaration should be preceded by an empty line<br>Line should be indented with spaces, not tabs<br>Selector `banner_content` should be written in lowercase with hyphens